USAGE.md

```diff
@@ -766,8 +766,9 @@ To log a message on `INFO` log level:
 logger.info("The current time is #{Time.now}")
 ```
 
-The default logger name for UI rules is `org.openhab.automation.jrubyscripting.script`.
-For file-based rules, it's based on the rule's ID, such as `org.openhab.automation.jrubyscripting.rule.myrule.rb:15`.
+The main logger prefix is `org.openhab.automation.jrubyscripting`.
+The default logger name for UI rules is `org.openhab.automation.jrubyscripting.script.rule_id`.
```

Comment:
It's not clear that rule_id is a placeholder here

Reply:
I've changed the wordings a bit.